cupy/fft/config.py

```diff
@@ -1,3 +1,23 @@
-
 global enable_nd_planning
```

Comment:
@grlee77 quick question: Do you recall why this needs to be `global`? Wouldn't normal module-level variables just work?

Reply:
I'm not sure why I had that there. I don't think it needs to be global.

Reply:
Ok, will remove it. Thanks.